src/components/PinButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {Pressable} from 'react-native';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import Icon from './Icon';
+import Tooltip from './Tooltip';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import iouReportPropTypes from '../pages/iouReportPropTypes';
+import * as Report from '../libs/actions/Report';
+import * as Expensicons from './Icon/Expensicons';
+import * as Session from '../libs/actions/Session';
+
+const propTypes = {
+    /** Report to pin */
+    report: iouReportPropTypes,
```

Comment:
I think this should use reportPropTypes instead because it's more general, and we pass more than just IOU reports to this component.